$MirOS$

	While the other examples require ewe-GTK+, this one
	works with command line too, so use the one bundled
	in the same subpackage.

--- examples/bottles/Makefile.orig	Thu Apr 10 17:18:01 2008
+++ examples/bottles/Makefile	Sun Oct  5 15:17:30 2008
@@ -8,7 +8,7 @@ CLEANFILES+=	${JOBJS}
 all: ${JOBJS}
 
 run:
-	ewe ${JPROG}
+	eweclne ${JPROG}
 
 .SUFFIXES: .class .java
 
